src/CSET/operators/constraints.py

```diff
@@ -205,6 +209,9 @@ def generate_area_constraint(
     -------
     area_constraint: iris.Constraint
     """
+    if lat_start == "None":
+        return iris.Constraint()
```

Comment:
```suggestion
    if lat_start is None:
        return iris.Constraint()
```
Only if the above suggested changed makes it actually a `None`.

Reply:
Discarding suggestion as doesn't work.